ps17/composer.json

```diff
@@ -13,16 +10,18 @@
         }
     },
     "require": {
+        "php": "^7.1",
+        "giggsey/libphonenumber-for-php": "^8.12",
         "gmponos/guzzle_logger": "^2.2",
+        "phpdocumentor/reflection-docblock": "^5.3",
+        "prestashop/module-lib-guzzle-adapter": "^1.0",
         "prestashop/module-lib-mbo-installer": "^2.0",
         "prestashop/module-lib-service-container": "^2.0",
         "prestashop/prestashop-accounts-installer": "^1.0",
-        "prestashop/module-lib-guzzle-adapter": "^1.0",
-        "giggsey/libphonenumber-for-php": "^8.12"
+        "symfony/property-access": "^2.8",
+        "symfony/serializer": "^2.8"
```

Comment:
Already bundled in PrestaShop Core dependencies so it will be available inside PrestaShop for the module. If you need it for tests or local, you can move it to dev dependencies instead.